### PR TITLE
Add 2.8.0/Tails 6 upgrade guide; bump version

### PR DESCRIPTION
**Pull request by eloquence**
## Status
Ready for review

## Description of Changes

- Standard upgrade boilerplate
- Step-by-step instructions to upgrade to Tails 6
- Pruned old upgrade guides

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/upgrade/2.7.0_to_2.8.0.rst [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+not possible.
+
+Follow our instructions to :ref:`update Tails manually <Update Tails Manually>`.
+
+Step 3: Apply SecureDrop-specific configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Boot up the updated workstation, connect to the Tor network, and run the
+following commands in a terminal: ::
+
+  cd ~/Persistent/securedrop
```
Comment (zenmonkeykstop) [inline docs/upgrade/2.7.0_to_2.8.0.rst:102]: Add an "sudo apt update" here before the setup command, to make sure the apt cache is up-to-date.

File: docs/upgrade/2.7.0_to_2.8.0.rst [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    git checkout 2.8.0
+
+.. important:: If you do see the warning "refname '2.8.0' is ambiguous" in the
+  output, we recommend that you contact us immediately at securedrop@freedom.press
+  (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
+
+Finally, run the following commands: ::
+
+  ./securedrop-admin setup
```
Comment (zenmonkeykstop) [inline docs/upgrade/2.7.0_to_2.8.0.rst:87]: `sudo apt update` here too before `setup`

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]

File: docs/upgrade/2.7.0_to_2.8.0.rst (hunk 1 above)
Comment (eloquence) [reply docs/upgrade/2.7.0_to_2.8.0.rst:103]: Done

File: docs/upgrade/2.7.0_to_2.8.0.rst (hunk 2 above)
Comment (eloquence) [reply docs/upgrade/2.7.0_to_2.8.0.rst:88]: Done

**Review by zenmonkeykstop — APPROVED** [review]

**Pull request merged**